src/components/Todo/Todo.js

```diff
@@ -1,8 +1,65 @@
-import React from 'react'
+import React, { useState } from 'react'
 import ListItem from 'components/ListItem/ListItem'
+import Input from 'components/Input/Input'
+import Button from 'components/Button/Button'
+import { FaEdit } from 'react-icons/fa'
+import { MdCancel } from 'react-icons/md'
 
-const Todo = ({ children, ...props }) => {
-  return <ListItem {...props}>{children}</ListItem>
+const Todo = ({ onUpdateText, id, value, children, ...props }) => {
+  const [editing, setEditing] = useState(false)
```

Comment:
`editing` se usa para determinar si se muestra el ícono de editar o no, y se activa / desactiva cuando se pasa el mouse por arriba. En realidad, lo que está detectando es eso, por lo que sería ideal ponerle un nombre que indice eso

```suggestion
  const [isMouseOver, setIsMouseOver] = useState(false)
```